modules/statics/src/coins.ts

```diff
@@ -7111,6 +7119,30 @@ export const coins = CoinMap.fromCoins([
     '0xe9e7cea3dedca5984780bafc599bd69add087d56',
     UnderlyingAsset['bsc:busd']
   ),
+  bscToken(
+    '2400c915-d610-45e7-bd7d-a028eb653b42',
+    'bsc:usdt',
+    'Tether USD',
```

Comment:
```suggestion
    'BSC-USD',
```
Should this one follow the naming: https://bscscan.com/address/0x55d398326f99059ff775485246999027b3197955

Reply:
@evalenti-simtlix - les go with what michael suggested here